package.json

```diff
@@ -22,6 +22,7 @@
   },
   "homepage": "https://github.com/userclouds/sdk-typescript#readme",
   "dependencies": {
+    "@userclouds/sdk-typescript": "^1.0.16",
```

Comment:
Oh this doesn't seem right

Reply:
@dstello https://github.com/userclouds/userclouds/commit/55366efdd539765a4a8c521c0bfe957285e52213#diff-74e06e119c867ba64dfe4e10a2f87887559d006092c39d9a2fde686627050052R25

Reply:
Yep, I'm going to look at that now